notebook/services/contents/handlers.py

```diff
@@ -122,9 +126,17 @@ def patch(self, path=''):
         model = self.get_json_body()
         if model is None:
             raise web.HTTPError(400, u'JSON body missing')
+        self.log.info(model)
         model = yield maybe_future(cm.update(model, path))
         validate_model(model, expect_content=False)
         self._finish_model(model)
+        self.log.info(model)
+        self.eventlog.record_event(
+            eventlogging_schema_fqn('contentsmanager-actions'), 1,
+            # FIXME: 'path' always has a leading slash, while model['path'] does not.
+            # What to do here for source_path? path munge manually? Eww
```

Comment:
Ew is right. 

It looks like we call `path.strip('/')` [in many places](https://github.com/jupyter/notebook/search?q=path.strip%28%27%2F%27%29&unscoped_q=path.strip%28%27%2F%27%29) across the notebook codebase to strip leading slash.